eng/Version.Details.xml

```diff
@@ -19,14 +19,10 @@
       <Sha>8000dd1a9ad9017e0cd1e9fd079c42b6dd7e6962</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
+    <!-- Don't declare a separate intermediate as xliff-tasks doesn't produce one. -->
```

Comment:
By including the SB metadata on this dependency, it implies the existence of an intermediate. As such, I'd adjust this comment to say something along the lines of "Don't declare a separate xliff-tasks intermediate as Arcade 8.0 does not have xliff intermediate declared".